AdminUi/apps/admin_ui/lib/core/modals/add_quota_dialog.dart

```diff
@@ -92,37 +92,37 @@ class _AddQuotaDialogState extends State<_AddQuotaDialog> {
                 value: _selectedMetric,
                 items: widget.availableMetrics.map((metric) => DropdownMenuItem(value: metric.key, child: Text(metric.displayName))).toList(),
                 onChanged: _saving ? null : (String? selected) => setState(() => _selectedMetric = selected),
-                decoration: const InputDecoration(
-                  border: OutlineInputBorder(),
-                  labelText: 'Metric*',
+                decoration: InputDecoration(
+                  border: const OutlineInputBorder(),
+                  labelText: '${context.l10n.metric}*',
                 ),
               ),
               Gaps.h24,
               TextField(
                 controller: _maxAmountController,
                 enabled: !_saving,
-                decoration: const InputDecoration(
-                  border: OutlineInputBorder(),
-                  labelText: 'Max Amount*',
-                  helperText: 'Only numbers greater or equal to 0 are valid.',
+                decoration: InputDecoration(
+                  border: const OutlineInputBorder(),
+                  labelText: '${context.l10n.maxEntities('Amount')}*',
```

Comment:
There are many places where you use this kind of pattern. What you forget is: `Amount` never gets translated here. I think you should not mix translated and non-translated text in this way.

Please go over the whole PR with this information, there are a bunch of occurences and I will not review until this is fixed.